src/main/java/taxi/controller/authentication/LogOutController.java

```diff
@@ -0,0 +1,18 @@
+package taxi.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(urlPatterns = "/logout")
```

Comment:
Should you move it to web.xml for a consistent approach in all controllers?(suggestion)